core/src/apps/bitcoin/addresses.py

```diff
@@ -24,10 +24,12 @@ def get_address(
     multisig: MultisigRedeemScriptType = None,
 ) -> str:
 
-    if (
-        script_type == InputScriptType.SPENDADDRESS
-        or script_type == InputScriptType.SPENDMULTISIG
-    ):
+    if script_type in [
+        InputScriptType.SPENDADDRESS,
+        InputScriptType.SPENDMULTISIG,
+        InputScriptType.SPENDSSRTX,
+        InputScriptType.SPENDSSGEN,
+    ]:
```

Comment:
This should be something like this:

`if script_type in [InputScriptType.SPENDADDRESS, InputScriptType.SPENDMULTISIG] or coin.decred and script_type in [InputScriptType.SPENDSSRTX, InputScriptType.SPENDSSGEN]`